pkg/uevent/match.go

```diff
@@ -56,7 +56,6 @@ var stageOneMatchers = []matchFn{
 	ueventFSUUIDMatcher,
 	// v1beta2 matchers
 	fsUUIDMatcher,
-	serialNumberMatcher,
```

Comment:
Why are we removing this `serialNumberMatcher` . IS it because , yesterday we saw that even if the device are not changed the serial number changes ?

Reply:
If `serialNumberMatcher`  is removed , then its corresponding function `serialNumberMatcher` and test `TestSerialNumberMatcher` must be removed too

Reply:
> Why are we removing this serialNumberMatcher 

this probe is not needed as we removed smart serial number probe

> If serialNumberMatcher is removed , then its corresponding function serialNumberMatcher and test TestSerialNumberMatcher must be removed too

such utility functions can be used to write supportability tools for upgrade. so, keeping such functions as is.

Reply:
actuall, we can remove. we don't need to rely on smart serial numbers even in future.